pwa/src/templates/templateParts/filters/verticalFilters/VerticalFiltersTemplate.module.css

```diff
@@ -16,10 +16,29 @@
   font-size: var(--web-app-size-md);
 }
 
+.trigger > *:not(:last-child) {
```

Comment:
The label and the icon are not aligned properly. An easy way to fix this would be:

```
.trigger {
  display: flex;
  align-items: center;
}
```